File: javascript/ordena.js
/**
 * Ordenar array utilizando bubbleSort simples
 */
function bubbleSort(arr)
{
    for(a=0,a_s=arr.length; a<a_s; a++)
    {
        for(b=a+1; b<a_s; b++)
        {
            if(arr[a]>arr[b])
            {
                var aux=arr[a];
                arr[a]=arr[b];
                arr[b]=aux;
            }
        }
    }
    return arr;
}

var array_teste = ['5','0','6','9','1','3','4','7','8','2'];
bubbleSort(array_teste);

File: python/ordena.py
# -*- coding: utf-8 -*-

# função bubblesort: algoritmo de ordenação simples 
def bubblesort(lista):
    for a in range(len(lista)-1,0,-1):
        for b in range(a):
            if lista[b] > lista[b+1]:
                aux = lista[b]
                lista[b] = lista[b+1]
                lista[b+1] = aux

# inicializar
lista = ['5','0','6','9','1','3','4','7','8','2']
bubblesort(lista)
print(lista)